src/bootstrap/bootstrap.py

```diff
@@ -312,6 +312,8 @@ def default_build_triple(verbose):
         # non-standard string (e.g. gnuwin32 tools returns `windows32`). In
         # these cases, fall back to using sys.platform.
         return 'x86_64-pc-windows-msvc'
+    elif kernel == 'AIX':
```

Comment:
Would it make more sense to add AIX to the kerneltype map above?

Reply:
I've added comment to describe the reason :).